README.md

```diff
@@ -6,15 +6,20 @@ This repository includes a Continuous Integration system that will build automat
 [Pull Request](https://github.com/conan-io/conan-center-index/pulls).
 
 
-### You can learn how to contribute new recipes and how this build service works in [the wiki of the project](https://github.com/conan-io/conan-center-index/wiki).
+### Documentation
+
+* If you want to learn how to contribute new recipes, please read [docs/how_to_add_packages.md](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md).
+* Most common questions are listed in [docs/faqs.md](https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md).
+* The conan-center hook errors reported by CCI Bot can be found in the [docs/error_knowledge_base.md](https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md).
```

Comment:
I believe using relative links should work https://github.blog/2013-01-31-relative-links-in-markup-files/
```suggestion
* If you want to learn how to contribute new recipes, please read [docs/how_to_add_packages.md](docs/how_to_add_packages.md).
* Most common questions are listed in [docs/faqs.md](docs/faqs.md).
* The conan-center hook errors reported by CCI Bot can be found in the [docs/error_knowledge_base.md](docs/error_knowledge_base.md).
```

Reply:
that's great!